File: HackerRank Solutions/10 Days of Statistics/Day 5 Poisson Distribution II.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner scan = new Scanner(System.in);
        double A = scan.nextDouble();
        double B = scan.nextDouble();
        scan.close();
        double dailyCostA = 160 + (40 * (A + (A * A)));
        double dailyCostB = 128 + (40 * (B + (B * B)));
        System.out.format("%.3f%n", dailyCostA);
        System.out.format("%.3f%n", dailyCostB);
    }
}

File: HackerRank Solutions/10 Days of Statistics/Day 5 Poisson Distribution II/Day 5 Poisson Distribution II.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <iomanip>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    double A, B;
    cin >> A >> B;
    double dailyCostA = 160 + (40 * (A + (A * A)));
    double dailyCostB = 128 + (40 * (B + (B * B)));
    cout << fixed << setprecision(3) << dailyCostA << "\n" << dailyCostB << endl;
    return 0;
}